pages/common/kubectl-expose.md

```diff
@@ -0,0 +1,16 @@
+# kubectl edit
+
+> Expose a resource as a new Kubernetes service.
+> More information: <https://kubernetes.io/docs/reference/generated/kubectl/kubectl-commands#expose>.
+
+- Create a service for a resource, to serve from container port to node port:
+
+`kubectl expose {{resource_type}} {{resource_name}} --port={{node_port}} --target-port={{container_port}}`
+
+- Create a service for a resource identified by defination file, to serve from container port to node port:
+
+`kubectl expose -f {{path/to/file.yml}} --port={{node_port}} --target-port={{container_port}}`
+
+- Create a service with a name, to serve to a node port which will be same for container port:
+
+`kubectl expose {{resource_type}} {{resource_name}} --port={{node_port}} --name={{service_name}}`
```

Comment:
```suggestion
- Create a service with a name:

`kubectl expose {{resource_type}} {{resource_name}} --name={{service_name}}`
```

Since there are two other examples using ports, I removed this parameter here.

Reply:
The `kubectl expose` command allows us to create a service that exposes a port from the container. By default, it chooses a port automatically if we don't specify one. However, this may not be desirable in some cases, because other services may rely on a fixed port. Therefore, I would suggest keeping the port option here, even though it is not strictly necessary.

Reply:
Okay sure :) I'll go ahead and approve